docs/en/roadmap/Introduction.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: global
+title: Alluxio Contribution Roadmap
+group: Roadmap
+priority: 0
+---
+# Alluxio Contribution Roadmap (In Progress)
+
+We warmly welcome you to the Alluxio community. We are excited about your contributions and engagement with our project!
+This guide aims to give you step-by-step instructions on how to get started becoming a contributor to the Alluxio open source project
+and become a qualified independent contributor eventually.
+
+This guide is not a quick start guide but provides a comprehensive roadmap of how to grow from a person beginning with only passion and motivation 
+to a qualified independent contributor that can make useful contributions to open source projects like Alluxio.
+
+If you are a student, considering it as an internship with industry training. 
+If you are a new grad, considering it as the onboarding guide provided by your company to train you to be an independent contributor. 
+If you want to contribute to an open-source project and expand your capability zone, this guide helps you to expose to an industry-level project while learning java, git, distributed system, etc. 
```

Comment:
this guide introduces you to an industry-level project while learning about java, git, distributed systems, and much more.